File: MEVN-App/backend/models/Account.js
const mongoose = require('mongoose');
const schema = new mongoose.Schema({
  firstName: String,
  email: { type: String, unique: true },
  password: String,
  memberType: { type: String, enum: ['donor', 'beneficiary'] },
  tokens: { type: Number, default: 0 }
});
module.exports = mongoose.model('Account', schema);

File: RALS/backend/src/models/account.rs
use serde::{Deserialize, Serialize};

#[derive(Serialize, Deserialize, Debug, sqlx::FromRow)]
pub struct Account {
    pub id: Option<i64>,
    pub first_name: String,
    pub email: String,
    pub password: String,
    pub member_type: String,
    pub tokens: Option<i64>,
}

#[derive(Deserialize)]
pub struct LoginData {
    pub email: String,
    pub password: String,
}
